### PR TITLE
Add flags to the add volume command

### LINKED ISSUE #609: Configurable reclaimPolicy for adding a persistent-volume
The `pv.yaml` template has persistentVolumeReclaimPolicy hardcoded to 'Retain'. (see below)

https://github.com/apprenda/kismatic/blob/master/ansible/roles/persistent-volume/templates/pv.yaml#L12

I would recommend adding a flag to `kismatic storage add` to override this default.

Scenario: 
As every Persistent Volume (PV) I create is set to _Retain_ this means I have to manually wipe the contents of the PV every time I want to redeploy.

### PR DESCRIPTION
**Pull request by alexbrand**
Fixes #609 

Broken up into two commits to separate auto-generated documentation changes.

### CODE REVIEW

File: ansible/roles/persistent-volume/templates/pv.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ spec:
     storage: {{ volume_quota_gb }}Gi
   accessModes:
     - ReadWriteMany
```
Comment (dkoshkin) [inline ansible/roles/persistent-volume/templates/pv.yaml]: While we are in here do you think its worth externalizing this option also? A mismatch in PV/PVC here will cause it not to mount.

Comment (alexbrand) [inline ansible/roles/persistent-volume/templates/pv.yaml]: Yes, definitely. Good catch!

**Comment by dkoshkin** [comment]
LGTM

**Comment by alexbrand** [comment]
The storage test I wanted to see pass has passed in CI. Merging this in.

**Pull request merged**